python/exercise1/vowel_counter.py

```diff
@@ -2,17 +2,19 @@
 
```

Comment:
Recommend creating a separate script for num_consonants called "consonant_counter" or rename the file to "vowel_and_consonant_counter" to avoid confusion.